src/lib/schematics/address-form/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.html

```diff
@@ -94,7 +94,7 @@
       </div>
     </mat-card-content>
     <mat-card-actions>
-      <button mat-raised-button color="primary" type="submit">Submit</button>
+      <button mat-raised-button color="primary" type="submit" (click)="onSubmit()">Submit</button>
```

Comment:
Shouldn't this go onto the form? This is technically a `click` not a submit.

Reply:
Yeah, that would be better as it also triggers validation of the form and display of `mat-error`s.

Looking into that also exposed a problem with the way that the postal code validators were defined.